tests/Example/cookie-protected/cookies.txt

```diff
@@ -1 +1 @@
-127.0.0.1	TRUE	/	FALSE	0	hello	world
+127.0.0.1	TRUE	/	FALSE	10000000000000	hello	world
```

Comment:
So 0 means expire immediately and this was wrong? (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie)

Reply:
I'm not sure anymore why I did that, probably?

Reply:
I'm not sure about this either. /shrug